lessons/221/caddy/Caddyfile

```diff
@@ -1,37 +1,31 @@
-www-caddy.antonputra.pvt:443 {
+www-caddy.antonputra.pvt {
 	# Set this path to your site's directory.
 	root * /data/my-website
 
-    # Enable compression.
-    encode gzip
+	# Enable compression.
+	encode gzip 3
```

Comment:
This isn't right, it needs to be:

```
	encode {
		gzip 3
	}
```

Only simple encoder names can be on the argument line, if options are to be given to the encoder then it needs to go in subdirectives.

Reply:
Thanks @francislavoie, can you please PR?